packages/cli/src/helpers.ts

```diff
@@ -76,6 +77,10 @@ export function execPromise(command: string): Promise<string> {
 }
 
 export async function checkCannonVersion(currentVersion: string): Promise<void> {
+  if (!(await isConnectedToInternet())) {
+    console.log('You are offline so we dont check the latest version of cannon');
```

Comment:
this can just be a debug message rather than a full on message to the user

Reply:
Yeah you are right, done.